MessageCenter3/Models/Message/MessageRepository.cs

```diff
@@ -11,10 +11,6 @@ namespace MessageCenter3.Models
 {
     public class MessageRepository : IMessageRepository
```

Comment:
Why is the repository class located in the Models project(_MessageCenter3/Models/Message/MessageRepository.cs_)?

Reply:
Moved to DAL.Repositories namespace (and physical location respectively)